account/src/repositories/accountRepository.js

```diff
@@ -1,37 +1,38 @@
 
 import { MongoClient } from 'mongodb';
 
-async function getUsersCollection() {
-  const connectionURL = 'mongodb://jessica:123456789@mongodb:27017';
-  const connection = new MongoClient(connectionURL);
-  await connection.connect();
+console.log(process.env.DATABASE_URL);
+export const client = new MongoClient(process.env.DATABASE_URL);
 
-  const database = connection.db('accounts');
-  return database.collection('users');
+export function getUsersCollection(client) {
+  const db = client.db('accounts');
+  const usersCollection = db.collection('users');
+  return usersCollection;
 }
 
 export async function saveAccount(accounts) {
-  const collection = await getUsersCollection();
-  // promise
-  await collection.insertOne(accounts);
+  await client.connect();
```

Comment:
will you create a connection everytime you save an account?

Reply:
Had done as per the mentor's explanation but i changed the code structure for mvc. So I'm not using this function. 
I uploaded the change in the sprint 9.